docs/install/install-from-packages.md

```diff
@@ -237,64 +217,97 @@ mkdir my_project
 cd my_project
 ```
 
-Run `cookiecutter` to create a Plone project skeleton using the Cookiecutter {term}`cookiecutter-plone-starter` with the following command.
+Issue the following command to install or update `cookiecutter`, then run it to create a Plone project skeleton using the Cookiecutter {term}`cookiecutter-plone-starter`.
 
 ```shell
-cookiecutter https://github.com/collective/cookiecutter-plone-starter
+pipx run cookiecutter gh:collective/cookiecutter-plone-starter
 ```
 
 You will be presented with a series of prompts.
 You can accept the default values in square brackets (`[default-option]`) by hitting the {kbd}`Enter` key, or enter your preferred values.
 For ease of documentation, we will use the default values.
 
 ```console
-You've downloaded <path-to-cookiecutter>/cookiecutter-plone-starter before. Is it okay to delete and re-download it? [yes]: 
-project_title [Project Title]: 
-project_slug [project-title]: 
-description [A new project using Plone 6.]: 
-author [Plone Foundation]: 
-email [collective@plone.org]: 
-python_package_name [project_title]: 
-plone_version [6.0.0]: 
-volto_version [16.5.0]: 
-volto_generator_version [6.2.0]: 
-Select language_code:
-1 - en
-2 - de
-3 - es
-4 - pt-br
-Choose from 1, 2, 3, 4 [1]: 
-github_organization [collective]: 
-Select container_registry:
-1 - Docker Hub
-2 - GitHub
-Choose from 1, 2 [1]: 
-================================================================================
-Project Title generation
+% pipx run cookiecutter gh:collective/cookiecutter-plone-starter
+
+
+Cookiecutter Plone Starter 
 ================================================================================
-Running sanity checks
-  - Python: ✓
-  - Node: ✓
-  - yo: ✓
-  - Docker: ✓
-  - git: ✓
+
+Sanity checks
+--------------------------------------------------------------------------------
+  [1/5] Python: ✓
+  [2/5] Node: ✓
+  [3/5] yo: ✓
+  [4/5] Docker: ✓
+  [5/5] git: ✓
+
+Project details
+--------------------------------------------------------------------------------
+
+  [1/19] Project Title (Project Title): Plone Conference Website 2070
+  [2/19] Project Description (A new project using Plone 6.): 
+  [3/19] Project Slug (Used for repository id) (plone-conference-website-2070): 
+  [4/19] Project URL (without protocol) (plone-conference-website-2070.example.com): 
+  [5/19] Author (Plone Foundation): Elli
+  [6/19] Author E-mail (collective@plone.org): elli@plone.org
+  [7/19] Python Package Name (plone_conference_website_2070): 
+  [8/19] Volto Addon Name (volto-plone-conference-website-2070): 
+  [9/19] Choose a Python Test Framework
+    1 - pytest
+    2 - unittest
+    Choose from [1/2] (1): 
+  [10/19] Plone Version (6.0.8): 
+  [11/19] Should we use Volto Alpha Versions? (No): yes
+  [12/19] Volto Version (18.0.0-alpha.1): 
```

Comment:
Should we choose the installation of alphas or enter a stable version? 

```suggestion
  [12/19] Volto Version (18.0.0-alpha.1): 
```

Reply:
I chose explicitly the alpha, because this, from package, is in most cases setting up Plone for development, not for production.